src/axom/primal/geometry/Ray.hpp

```diff
@@ -102,7 +103,7 @@ class Ray
    * \brief Default Constructor. Does nothing.
    * \note Made private to prevent its use in application code.
    */
-  Ray() {};
+  Ray() = default;
```

Comment:
This could be made a public `delete`-ed constructor, which might convey intent better.